Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -316,7 +317,12 @@ export default class FilterComponent extends Vue {
                                 {{ filter.display }}
                             </b-form-checkbox>
                         </b-col>
-                        <b-col cols="4" align-self="end" class="text-right">
+                        <b-col
```

Comment:
This will create the rows even if they are empty. Instead the b-row should not be iteraterated if is not enabled. That way we avoid having to check for is enabled on the columns